redesign/cs-smg.html

```diff
@@ -0,0 +1,204 @@
+    <meta name="robots" content="index, follow, noarchive">
+    <meta name="base" content="https://www.dwyl.com">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Istok+Web|Montserrat:500" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="css/dwyl.css">
+    <link rel="shortcut icon" href="/img/favicon.ico">
+    <title>dwyl</title>
+  </head>
+  <body>
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NSP64J3"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <!-- End Google Tag Manager (noscript) -->
+    <nav class="w-100 dwyl-bg-dark-gray tc white relative fixed">
+      <input type="checkbox" id="burger" class="absolute top-2 right-1 dn">
+      <label for="burger" class="dn-ns pointer absolute top-1 right-1">
+        <i class="fa fa-bars white dib f2"></i>
+      </label>
+      <ul class="overflow-hidden menu db-ns w-100 list tc pl0 pt3 mv0 f3 fw3 f5-ns">
+        <li class="absolute top-1 static-ns ph4 mh2 fw3 di-ns pt1 pb3 pv3-ns">
+          <a href="/" class="white link">
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w1-plus"/>
+            dwyl
+          </a>
+        </li>
+        <li class="ph5 pt5 pb2 tl dn-ns"><a href="/" class="white link">Home</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/portfolio" class="white link">Portfolio</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/values" class="white link">Values</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/team" class="white link">Team</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/app" class="white link">App
+          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph5 di-ns pt2 pb3 tl pv0-ns"><a href="/#contact" class="white link">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="pt5 pb3 pb3-ns ph6-ns center tc shadow-3">
+      <img src="./img/common/science-museum-logo.jpg" alt="Science Museum Group logo" class="w4 pt4 mt4"/>
+      <h2 class="dwyl-light-gray f4">The Science Museum Group (SMG)</h2>
+      <h1 class="dwyl-dark-gray f3-plus w-90 tc center">Online Databank for Museum Collections</h1>
+    </section>
+
+    <section class="pt2 pb3 pv4-ns w-80 center">
+      <div class="w-40-ns dtc-ns v-top-ns">
+        <img src="./img/common/science-museum-screenshot1.png" alt="A screenshot of the project search page" class="w-60-ns pl5-ns">
+        <img src="./img/common/science-museum-screenshot2.png" alt="A presentation of the project on stage" class="w-60-ns pt4 pl5-ns dn db-ns">
```

Comment:
All `alt` text is great, but needs to be more explanatory and include key words like 'science museum' in each one - think SEO.